Conceptpower+Spring/src/main/java/edu/asu/conceptpower/servlet/rest/Concepts.java

```diff
@@ -47,6 +48,24 @@
 
     private static final Logger logger = LoggerFactory.getLogger(Concepts.class);
 
+    /**
+     * This method creates concepts as well as concept wrappers. This is decided
+     * based on the request parameter. If the request parameter contains the
+     * wrapper ids then concept is wrapped else the new concept is created.
+     * 
+     * Sample input
+     * 
+     * { "wordnetids":
+     * "WID-00450866-N-01-pony-trekking,WID-03981924-N-01-pony_cart",
+     * "synonymids" : " WID-02380464-N-01-polo_pony,WID-04206225-N-03-pony",
+     * "conceptlist" : "FirstList-1", "type" :
+     * "88bea1dc-1443-4296-8315-715c71128b01" "description" : "Description",
+     * "equalsTo" : "equals", "similarTo" : "similar" }
```

Comment:
this should have also a json example for regular adding a concept and make clear what json does what.